packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -76,11 +76,15 @@ export class GuxTimeZonePickerBeta {
   }
 
   private filterTimeZoneList(timeZoneList: GuxTimeZoneOption[]) {
-    const searchString = this.searchString;
+    const searchString = this.searchString.toLowerCase();
```

Comment:
Should we do a null check for `this.searchString` like so? `this.searchString?.toLowerCase();` Perhaps if `this.searchString` is ever null maybe we should check that a level higher than this. I see `this.searchString` gets set with this code: `this.searchString = event.detail;` on line 55. Perhaps we should do `this.searchString = event.detail || "";` instead?

Reply:
I dont think a `null` check would be required here as the `this.searchString` is set via the `gux-filter` which would mean a key has been pressed and the detail of that event would be assigned to `this.searchString` ?